### DISCUSSION TITLE
Pulling data from parent tables into nested tables

### DISCUSSION THREAD

**Discussion by Hajelarohit** [Budibase/budibase#6096] [Help]
Hi BB Team/community

I have three tables
Supplier > Contracts > GovSch
1 Supplier > Many Contract
1 Contract > Many GovSch

I have defined the supplier name in the Supplier table (topmost table), I can pull the supplier name in the Contract table using a formula. However it fails when i am trying to pull this into the GovSch (Governance Schedule) table. How can i achieve it?

https://vimeo.com/714530635/6cd392f78d

**Comment by melohagan** [comment]
Hi @Hajelarohit 

I believe this is a very similar scenario: https://github.com/Budibase/budibase/discussions/5191#discussioncomment-2498586

Please note the last line of that comment -> the first level formula as you had had to be **static**.
The second level one had to be **dynamic**.

So in other words what you had was correct, you just need to change the `{{ Contract.0.Supplier Name }}` from a **static** filter to a **dynamic** filter.

As you can imagine this would break with another level of relationships. Given the more complex nature of your database schema, I would recommend using an external RDBMS.
